Libraries/Network/NetInfo.js

```diff
@@ -196,11 +202,18 @@ const NetInfo = {
     _subscriptions.delete(handler);
   },
 
+  /*
+   * Returns a promise that returns any of the connectivity types listed above.
+   */
   fetch(): Promise {
     return RCTNetInfo.getCurrentConnectivity().then(resp => resp.network_info);
   },
 
   isConnected: {
```

Comment:
Those won't show on the website as it doesn't extract methods from a nested object :( Still good to have the comments.